homework-submission/my-project/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "my-project",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^16.5.2",
+    "react-dom": "^16.5.2",
+    "react-scripts": "2.0.5"
```

Comment:
Two things are missed when I opened the application that stopped it from working:
 - react-datepicker
 - react-icons

So Make sure you've added `--save` flag after the `npm install` if you have a version of `npm` below the `5`th
The errors says:
 - You have a use of `react-datepicker` but you didn't include it within your project.
 - You have a use of `react-icons` but you didn't include it within your project.

<img width="732" alt="screenshot 2018-11-15 at 04 15 53" src="https://user-images.githubusercontent.com/25272822/48529423-5218e580-e893-11e8-8d15-68aa74098821.png">

<img width="734" alt="screenshot 2018-11-15 at 04 14 07" src="https://user-images.githubusercontent.com/25272822/48529422-51804f00-e893-11e8-8ead-b55c54d7e843.png">